.teamcity/src/main/kotlin/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -182,10 +182,6 @@ fun applyTestDefaults(
     daemon: Boolean = true,
     preSteps: BuildSteps.() -> Unit = {} // the steps before runner steps
 ) {
-    if (os == Os.MACOS) {
```

Comment:
BTW mac agents are not in Hetzner but at Ireland, but it's fine to also use `repo.grdev.net` as well.